scripts/obtain/dotnet-install.sh

```diff
@@ -104,11 +108,15 @@ get_current_os_name() {
                     echo "ubuntu.16.10"
                     return 0
                     ;;
+                "alpine.3.4.3")
```

Comment:
Should we modify this script to emit the read value, except for the cases where we don't wnt, like ubuntu.14.04, rhel and now this alpine one?